crypto/crmf/crmf_asn.c

```diff
@@ -43,7 +43,7 @@ ASN1_NDEF_SEQUENCE(CMS_EnvelopedData) = {
 /* explicit #includes not strictly needed since implied by the above: */
 #include <openssl/crmf.h>
 
-#if OPENSSL_VERSION_NUMBER <= 0x30200000L
+#if OPENSSL_VERSION_NUMBER <= 0x930200000L
```

Comment:
Is this version number intentional?

Reply:
Good catch - yes :smile:

I wanted to keep the position and more or less the contents of these conditions, while disabling them.
I've just improved this.